docs/usage.md

```diff
@@ -670,3 +802,36 @@ OPTIONS=”—default-ulimit nofile=65535:65535"
 Re-start your session.
 
 Note that the way to increase the open file limit in your system may be slightly different or require additional steps.
+
+#### Cannot delete work folder when using docker + Spark
+
+Currently, when running spark-based tools in combination with docker, it is required to set `docker.userEmulation = false`. This can unfortunately causes permission issues when `work/` is being written with root permissions. In case this happens, you might need to configure docker to run without `userEmulation` (see [here](https://github.com/Midnighter/nf-core-adr/blob/main/docs/adr/0008-refrain-from-using-docker-useremulation-in-nextflow.md)).
+
+### How to handle UMIs
+
+Sarek can process UMI-reads, using [fgbio](http://fulcrumgenomics.github.io/fgbio/tools/latest/) tools.
+
+In order to use reads containing UMI tags as your initial input, you need to include `--umi_read_structure [structure]` in your parameters.
+
+This will enable pre-processing of the reads and UMI consensus reads calling, which will then be used to continue Sarek workflow. It is then recommended to skip an additional duplicate marking and base quality score recalibration. See [`--skip_tools`].
```

Comment:
@lescai added the last part about skipping of bqsr and md based on our slack messages. Have no other sources for it though. but this is correct, right?

Reply:
argh yes I forgot that :) good catch! this is actually very important, thanks for adding it!

Reply:
added a vague statement now to at least have users think about it